### PR TITLE
release.md rename beta releases to next

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

File: aio/content/guide/releases.md [hunk 1]
```diff
@@ -54,17 +54,13 @@ See [Keeping Up-to-Date](guide/updating "Updating your projects") for more infor
...
--->
+* **Next:** The release that is under active development and testing. The next release is indicated by a release tag appended with the  `-next` identifier, such as  `8.1.0-next.0`.
 
-* **Beta:** A release that is under active development and testing. A Beta release is indicated by a release tag appended with the  `beta` identifier, such as  `8.0.0-beta.0`.
+* **Release candidate:** A release that is feature complete and in final testing. A release candidate is indicated by a release tag appended with the `-rc` identifier, such as version `8.1.0-rc.0`.
 
-* **Release candidate:** A release that is feature complete and in final testing. A release candidate is indicated by a release tag appended with the `rc` identifier, such as version `8.1.0-rc`.
-
-The next version of the documentation is available at [next.angular.io](https://next.angular.io). This includes any documentation for Beta or Release Candidate features and APIs.
+The `next` and `rc` version of the documentation is available at [next.angular.io](https://next.angular.io).
```
Comment (gkalpak) [inline aio/content/guide/releases.md]: Well...RCs are not deployed. Only `-next` and stable :grin:

File: aio/content/guide/releases.md [hunk 2]
```diff
@@ -84,9 +80,9 @@ In general, you can expect the following release cycle:
 
 * 1-3 minor releases for each major release
 
-* A patch release almost every week
+* A patch release and pre-release (`next` and `rc`) builds almost every week
```
Comment (kara) [inline aio/content/guide/releases.md]:
```suggestion
* A patch release and pre-release (`next` and `rc`) build almost every week
```

File: aio/content/guide/releases.md (hunk 1 above)
Comment (kara) [inline aio/content/guide/releases.md]:
```suggestion
The `next` and `rc` version of the documentation are both available at [next.angular.io](https://next.angular.io).
```

**Review by kara — APPROVED** [review]
LGTM with nits

File: aio/content/guide/releases.md (hunk 1 above)
Comment (IgorMinar) [reply aio/content/guide/releases.md]: not at the same time, which is what your change implies.

File: aio/content/guide/releases.md (hunk 2 above)
Comment (IgorMinar) [reply aio/content/guide/releases.md]: sgtm

File: aio/content/guide/releases.md (hunk 1 above)
Comment (IgorMinar) [reply aio/content/guide/releases.md]: I clarified it more

File: aio/content/guide/releases.md [hunk 3]
```diff
@@ -54,17 +54,13 @@ See [Keeping Up-to-Date](guide/updating "Updating your projects") for more infor
...
--->
+* **Next:** The release that is under active development and testing. The next release is indicated by a release tag appended with the  `-next` identifier, such as  `8.1.0-next.0`.
 
-* **Beta:** A release that is under active development and testing. A Beta release is indicated by a release tag appended with the  `beta` identifier, such as  `8.0.0-beta.0`.
+* **Release candidate:** A release that is feature complete and in final testing. A release candidate is indicated by a release tag appended with the `-rc` identifier, such as version `8.1.0-rc.0`.
 
-* **Release candidate:** A release that is feature complete and in final testing. A release candidate is indicated by a release tag appended with the `rc` identifier, such as version `8.1.0-rc`.
-
-The next version of the documentation is available at [next.angular.io](https://next.angular.io). This includes any documentation for Beta or Release Candidate features and APIs.
+The latest `next` or `rc` pre-release version of the documentation is available at [next.angular.io](https://next.angular.io).
```
Comment (gkalpak) [inline aio/content/guide/releases.md]: FWIW, what I meant in https://github.com/angular/angular/pull/35276#discussion_r376927683 is that RCs are technically _NOT_ deployed anywhere.

Only the stable and master branches are deployed to `angular.io` and `next.angular.io`. Before we cut an RC, a new branch is created (so RCs are never cut from master). During RC periods, the stable branch is deployed to `angular.io` and master is deployed to `next.angular.io`. Master is close to the RC, but they are not necessarily identical.

(Probably not worth bothering with these details here (esp. given master will usually be close to the RC branch) - just clarifying what happens.)

**Pull request closed without merging**